app/loading-status-service/loading-status.service.ts

```diff
@@ -0,0 +1,29 @@
+import {Injectable} from "@angular/core";
+import {NavController, Loading, LoadingOptions} from "ionic-angular/index";
+import {LoadingStatus} from "../loading-status/loading-status";
+
+@Injectable()
+export class LoadingStatusService {
+
+    private loadingOptions: LoadingOptions;
+
+    constructor() {
+        this.loadingOptions = {};
```

Comment:
FYI - If we want to change any of the look-and-feel of the Loading mask, this is the single place in the app where we can do it.